catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -388,6 +429,23 @@ public void destroy() {
         }
     }
 
+    private void logRegistryMetacards(String message, Collection<Metacard> mcards) {
```

Comment:
Maybe make this more general by adding the logging level as a parameter? May not make sense if you'll never dump metacard collections at other than DEBUG